src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+
+    public Figure randomFigure() {
```

Comment:
```suggestion
    public Figure getRandomFigure() {
        ColorSupplier colorSupplier = new ColorSupplier();
```
use this variable to call the method

Reply:
done